src/manipulation.js

```diff
@@ -292,13 +292,13 @@ jQuery.extend( {
 						}
 					}
 
-					// Support: Chrome <= 35-45+
+					// Support: Chrome <=35 - 45+
 					// Assign undefined instead of using delete, see Data#remove
 					elem[ dataPriv.expando ] = undefined;
 				}
 				if ( elem[ dataUser.expando ] ) {
 
-					// Support: Chrome <= 35-45+
+					// Support: Chrome <=35 - 45+
```

Comment:
@gibson042 for this rule to became official, you gotta submit pr to the contribute repo. Wouldn't want contributors to follow invisible guidelines